openpype/hosts/max/plugins/create/create_review.py

```diff
@@ -33,7 +34,7 @@ def create(self, subset_name, instance_data, pre_create_data):
             pre_create_data)
 
     def get_instance_attr_defs(self):
-        image_format_enum = ["exr", "jpg", "png"]
+        image_format_enum = ["exr", "jpg", "png", "tga"]
```

Comment:
How come this `CreateReview` format enum is smaller than the `ExtractReview` `image_exts` variable? Are we just not exposing all options?

Reply:
Some of the options are not supported in Max so that's why I didn't expose all